model.go

```diff
@@ -167,6 +168,19 @@ type Instrument struct {
 
 type Protocol string
 
+type FTPConfig struct {
+	ID               uuid.UUID
+	InstrumentId     uuid.UUID
+	Username         string
+	Password         string
+	RemotePath       string
```

Comment:
RemotePath and ResultRemotePath are not the most describing names 
stg with opposing keywords would be preferable, like  OrderPath/ResultPath or ImportPath/ExportPath

i'd recommend separate filemask, filesuffix for the orders/results or import/export

Reply:
pls add createdAt too

Reply:
both done